immutable_tree.go

```diff
@@ -190,10 +190,15 @@ func (t *ImmutableTree) Get(key []byte) ([]byte, error) {
 		}
 
 		if fastNode == nil {
+			latestVersion, err := t.ndb.getLatestVersion()
```

Comment:
Note that `getLatestVersion()` will not return `t.dndb.latestVersion` if it is 0. I'm not sure if this is a problem or not. We could also add a simple getter that only returns `t.ndb.latestVersion` directly.

Reply:
I see there is already an exiting method `ImmutableTree::isLatestVersion` which does what my suggestion does, maybe we should just call that?

Reply:
@cool-develope I did the change as you suggested using a new `safeGetLatestVersion` method instead.